controllers/Articles.controller.js

```diff
@@ -0,0 +1,20 @@
+const {fetchArticles} = require('../models/Articles.model')
+const {countCommentsByArticleID} = require("../models/Comments.model")
+
+exports.getArticles = (req, res, next)=>{
+    fetchArticles()
+    .then((articles)=>{
+        const commentPromises = []
+        articles.forEach((article)=>{
+            commentPromises.push(countCommentsByArticleID(article.article_id))
+        })
+        Promise.all(commentPromises).then((commentCounts)=>{
+            articles.map((article, index)=>{
+                article.comment_count = commentCounts[index]
+                delete article.body})
+            res.status(200).send({'articles': articles})
```

Comment:
Don't know if setting the object {'articles': articles} within the articles.controller.js is best practice, but seemed far more cumbersome to do in the model when relying on data from the commentCounts promise